docs/Data/Lists/Length.md

```diff
@@ -0,0 +1,24 @@
+# `Length(list)`
+
+Gets the length of the `list`; otherwise, defaults to `0`.
```

Comment:
Is the second half of the sentence needed (semicolon and on)?

Reply:
The default is to acknowledge that the `list` may be `null`.